Tests/Resources/Fixtures/config/full.xml

```diff
@@ -44,9 +44,11 @@
                 <my_service type="service" service="my_auth_service"/>
             </authentication>
             <cache cache-pool="my_cache_pool" stream-factory="my_other_stream_factory">
-                <config default-ttl="42">
+                <config default-ttl="42" cache-lifetime="2592000" hash-algo="sha1" cache-key-generator="null">
                     <respect-response-cache-directives>X-Foo</respect-response-cache-directives>
                     <respect-response-cache-directives>X-Bar</respect-response-cache-directives>
+                    <methods>GET</methods>
+                    <methods>HEAD</methods>
```

Comment:
this is wrong. The XML tag should be named `method` and we need to have the call `->fixXmlConfig('method')` on the parent node.

Prototyped nodes work a bit special in XML, due to the way child tags are handled when parsing XML in SimpleXML (if you put a single method with your current code, you will not get an array).

thus, the tag name will be better with a singular